scripts/verify-coverage.js

```
@@ -0,0 +1,63 @@
+#!/usr/bin/env node
+
+/**
+ * This script verifies that test coverage meets the required thresholds.
+ * It's used in the pre-commit hook to ensure code quality.
+ */
+
+const fs = require('fs');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)